conf/cassandra.yaml

```diff
@@ -298,6 +298,16 @@ partitioner: org.apache.cassandra.dht.Murmur3Partitioner
 # containing a CDC-enabled table if at space limit in cdc_raw_directory).
 cdc_enabled: false
 
+# Specify whether writes to the CDC-enabled tables should be blocked when CDC data on disk has reached to the limit.
+# When setting to false, the writes will not be blocked and the oldest CDC data on disk will be deleted to
+# ensure the size constraint. The default is true.
+# cdc_block_writes: true
+
+# Specify whether enable write path for CDC during internodes streaming.
+# When enabled, CDC data streamed to the destination node will be written into commit log first. When setting to false,
+# the CDC data will be written into SSTables just the same as normal streaming. The default is true.
+# write_path_for_cdc_enabled: true
```

Comment:
Good point. I see how it is confusing now, since the normal write path must produce CDC data when enabled. 
Regarding `cdc_on_streaming_enabled`, I am a bit worried that it could be confused with streaming CDC to downstream consumer... And the name of the second option is probably too long, preferably a short one.  

Essentially, it is a repair mechanism for CDC data. How about `cdc_repair_enabled`?

Reply:
That makes it sound like it repairs cdc data. What about `cdc_on_repair_enabled`?